moment/moment-node.d.ts

```diff
@@ -346,6 +346,8 @@ declare namespace moment {
         min(date: Moment | string | number | Date | any[]): Moment;
         min(date: string, format: string): Moment;
 
+        parsingFlags(): any;
```

Comment:
@davidlipo have you tried using this field? can you try and document the structure of the return object?